src/screen/Polkaassembly/PolkaassemblyDiscussionDetail.tsx

```diff
@@ -0,0 +1,173 @@
+          <Text category="c1"> {moment(post.created_at).fromNow()}</Text>
+        </View>
+        <View style={styles.content}>
+          <Text style={styles.contentText}>{post.content?.trim() ?? ''}</Text>
+        </View>
+        <View style={styles.reactionRow}>
+          {post.comments.length ? (
+            <>
+              <View style={globalStyles.rowAlignCenter}>
+                <Icon name="message-circle-outline" style={globalStyles.icon15} fill="#ccc" animation="pulse" />
+                <Padder scale={0.3} />
+                <Text category="label" appearance="hint">
+                  {post.comments.length} comments
+                </Text>
+              </View>
+              <Padder scale={1} />
+            </>
+          ) : null}
+          <Text category="c1" appearance="hint">
+            👍 {post.likes.aggregate.count}{' '}
+          </Text>
+          <Text category="c1" appearance="hint">
+            {' '}
+            👎 {post.dislikes.aggregate.count}
+          </Text>
+        </View>
+        <View style={styles.commentsContainer}>
+          {post.comments.map((comment) => (
+            <View key={comment.id} style={styles.comment}>
+              <View style={[styles.commentAuthorIcon, {backgroundColor: theme['text-basic-color']}]}>
+                <Text category="h6" appearance="alternative">
+                  {comment.author?.username?.substr(0, 1).toUpperCase()}
+                </Text>
+              </View>
+              <View style={styles.commentRightSide}>
+                <View style={styles.commentHeader}>
+                  <Text category="c2" numberOfLines={1} style={styles.commentAuthor}>
+                    {comment.author?.username ?? ''}
+                  </Text>
+                  <Text category="c1"> commented </Text>
+                  <Text category="c2">{moment(comment.created_at).fromNow()}</Text>
+                </View>
+                <Padder scale={0.5} />
+                <Text category="c1">{comment.content.trim()}</Text>
+                <Padder scale={0.5} />
+                <View style={globalStyles.rowAlignCenter}>
+                  <Text category="c1" appearance="hint">
+                    👍 {comment.likes.aggregate.count}{' '}
+                  </Text>
+                  <Text category="c1" appearance="hint">
+                    {' '}
+                    👎 {comment.dislikes.aggregate.count}
+                  </Text>
+                </View>
+              </View>
+            </View>
+          ))}
+        </View>
+        <View style={styles.footer}>
+          <Icon style={globalStyles.icon15} name="info-outline" fill={theme['text-hint-color']} />
+          <Text appearance="hint" category="c1">
+            {' '}
+            To comment, like or subscribe please{' '}
+          </Text>
+          <TouchableOpacity onPress={() => Linking.openURL(externalURL)}>
+            <Text appearance="hint" category="c2">
+              login
```

Comment:
If the user logs in, does it keep the session alive when navigating back and forth in the app?

Reply:
yes the session stays active on polkassembly